controllers/helper_test.go

```diff
@@ -0,0 +1,21 @@
+package controllers
```

Comment:
this file `controllers/helper_test.go` should be in the `utils` directory

Reply:
done